src/classes/PS_CommitmentRequest.cls

```diff
@@ -311,12 +367,21 @@ public inherited sharing class PS_CommitmentRequest {
                 return this;
             }
 
-            this.firstName = organization.Name;
-            this.lastName = organization.Name;
+            this.firstName = getMaxLenName(organization.Name);
+            this.lastName = getMaxLenName(organization.Name);
 
             return this;
         }
 
+        /**
+         * @description Return max number of character allowed for the first/last name in Elevate
+         * @param name First or Last Name
+         * @return String
+         */
+        private String getMaxLenName(String name) {
```

Comment:
```suggestion
        private String trimNameField(String name) {
```

Reply:
Done, I was going bak and forth with naming this one. `trim` keyword was a part of many variations :)